docs/user_guide/create-feature-toggle.md

```diff
@@ -0,0 +1,41 @@
+---
+id: create_feature_toggle
+title: Create a feature toggle
+---
+
+In this guide you will learn how to create your first feature toggle using Unleash. Login to your instance.
+
+## Step 1: Click the “create feature toggle” button
+
+The first time you log-in to your Unleash instance you will see an empty list of feature toggles. In order to create a new feature toggle you have to click the “create feature toggle” button
+
+![Create a feature toggle](../assets/create_toggle_1.png)
+
+## Step 2: Create Feature toggle
+
+After clicking the “create feature toggle button you will be presented with a form for creating a new feature toggle. You will need to define a few fields before you can actually complete the new feature toggle. 
+
+- **Name** – Must be unique across all your feature toggle. The name must also follow a URL friendly format. Can not be changed. 
+- **Description** – A good description makes it easier for other members on your team to understand why this feature toggle exists. 
+- **Enabled** – Whether the feature toggle should be enabled or disabled. If the feature toggle is disabled, activation strategy configurations will not be evaluated. 
+- **Activation** strategies –  A list of on or more activation strategies. An activation strategy is used to enable the feature toggle to a subset, or all, of your users. 
+
+In the example below we have chosen to configure the default activation strategy, which means that the feature will be enabled for everyone.
+
+![Create a feature toggle](../assets/create_toggle_2.png)
+
+## Step 3: Congratulations, you have now created your first feature toggle!
+
+The toggle is now created and ready to be used. The toggle does not have any metrics because  it is not used by any applications, yet!
```

Comment:
```suggestion
The toggle is now created and ready to be used. The toggle does not have any metrics because it is not used by any applications, yet!
```